3_core_core_analysis/1_core_core_relationships_across_strains.py

```diff
@@ -244,11 +249,10 @@ def compare_gene_relationships(gene_mapping_dict, mapping_to, pao1_corr, pa14_co
 
 # ## Plots
 
-# +
-# Add label for most and least stable genes
+# Add label for most and least stable genes using threshold cutoff
 pao1_corr_df["label"] = ""
 pa14_corr_df["label"] = ""
-pao1_corr_df.loc[
+"""pao1_corr_df.loc[
```

Comment:
Do you want to keep this commented-out code?

Reply:
No this isn't needed anymore